fdbserver/ClusterController.actor.cpp

```diff
@@ -2538,6 +2538,7 @@ ACTOR Future<Void> workerAvailabilityWatch(WorkerInterface worker,
 				cluster->removedDBInfoEndpoints.insert(worker.updateServerDBInfo.getEndpoint());
 				cluster->id_worker.erase(worker.locality.processId());
 				cluster->updateWorkerList.set(worker.locality.processId(), Optional<ProcessData>());
+				TraceEvent("ClusterFailedWorker", cluster->id).detail("WorkerProcessId", worker.locality.processId());
```

Comment:
To be constent with https://github.com/apple/foundationdb/blob/bc81b32c443387073620b9dbffee0cf08f8561ad/fdbserver/ClusterController.actor.cpp#L2767, maybe name it `ClusterControllerWorkerFailed|Removed`.

Also, I think we should also add the network address of worker and processClass.